lib/wasmer-types/src/entity/secondary_map.rs

```diff
@@ -279,6 +281,21 @@ where
     }
 }
 
+impl<K, V> MemoryUsage for SecondaryMap<K, V>
+where
+    K: EntityRef,
+    V: Clone + MemoryUsage,
+{
+    fn size_of_val(&self, tracker: &mut dyn MemoryUsageTracker) -> usize {
+        mem::size_of_val(self)
+            + self
+                .elems
+                .iter()
+                .map(|value| value.size_of_val(tracker) - mem::size_of_val(value))
+                .sum::<usize>()
+    }
+}
+
```

Comment:
I think you probably also need to do `self.default.size_of_val(tracker)`, but that's a small nit pick

Reply:
:+1: